Failing to find the root entities

With the latest tck runner setup the repository beans are being injected, and hibernate is configured, but I'm failing to see the tck entity tables being created, and when I run one the `StandaloneEntityTests#ensureNaturalNumberPrepopulation` test from the command line, there is no 

```bash
starksm@Scotts-Mac-Studio runner % mvn test -Dtest=org.hibernate.ee.jakarta.data.tck.StandaloneEntityTests#ensureNaturalNumberPrepopulation
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------< org.hibernate:hiberate-data-tck-runner >---------------
[INFO] Building Hibernate Jakarta Data TCK Runner 1.0
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- resources:3.3.1:resources (default-resources) @ hiberate-data-tck-runner ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] skip non existing resourceDirectory /Users/starksm/Dev/Jakarta/hibernate-data-tck/runner/src/main/resources
[INFO] 
[INFO] --- compiler:3.13.0:compile (default-compile) @ hiberate-data-tck-runner ---
[INFO] No sources to compile
[INFO] 
[INFO] --- resources:3.3.1:testResources (default-testResources) @ hiberate-data-tck-runner ---
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
[INFO] Copying 4 resources from src/test/resources to target/test-classes
[INFO] 
[INFO] --- compiler:3.13.0:testCompile (default-testCompile) @ hiberate-data-tck-runner ---
[INFO] Nothing to compile - all classes are up to date.
[INFO] 
[INFO] --- surefire:2.22.2:test (default-test) @ hiberate-data-tck-runner ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.hibernate.ee.jakarta.data.tck.StandaloneEntityTests
HibernateStandaloneExtension.beforeAll
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.997 s <<< FAILURE! - in org.hibernate.ee.jakarta.data.tck.StandaloneEntityTests
[ERROR] ensureNaturalNumberPrepopulation  Time elapsed: 0.983 s  <<< ERROR!
jakarta.data.exceptions.DataException: org.hibernate.query.sqm.UnknownEntityException: Could not resolve root entity 'NaturalNumber'
Caused by: org.hibernate.query.sqm.UnknownEntityException: Could not resolve root entity 'NaturalNumber'
```

This is coming from the test calling the numbers.countBy() method, which is hitting the following `NaturalNumbers_#countBy()` generated method:

```java
package ee.jakarta.tck.data.framework.read.only;
...
@RequestScoped
@Generated("org.hibernate.processor.HibernateProcessor")
public class NaturalNumbers_ implements NaturalNumbers {
...
	@Override
	public long countBy() {
		try {
			return session.createSelectionQuery(COUNT_BY, long.class)
				.getSingleResult();
		}
		catch (NoResultException exception) {
			throw new EmptyResultException(exception);
		}
		catch (NonUniqueResultException exception) {
			throw new jakarta.data.exceptions.NonUniqueResultException(exception);
		}
		catch (PersistenceException exception) {
			throw new DataException(exception);
		}
	}
```

The tck run.log which has debugging cranked up is attached:
[DataTCK00.log](https://github.com/jakartaredhat/hibernate-data-tck/files/14814545/DataTCK00.log)

I don't get this, I get:

```
[INFO] Running org.hibernate.ee.jakarta.data.tck.StandaloneEntityTests
HibernateStandaloneExtension.beforeAll
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.862 s <<< FAILURE! - in org.hibernate.ee.jakarta.data.tck.StandaloneEntityTests
[ERROR] ensureNaturalNumberPrepopulation  Time elapsed: 0.809 s  <<< ERROR!
org.jboss.weld.exceptions.DeploymentException: 
WELD-001408: Unsatisfied dependencies for type MultipleEntityRepo with qualifiers @Default
  at injection point [BackedAnnotatedField] @Inject ee.jakarta.tck.data.standalone.entity.EntityTests.shared
  at ee.jakarta.tck.data.standalone.entity.EntityTests.shared(EntityTests.java:0)
```

> [ERROR] ensureNaturalNumberPrepopulation  Time elapsed: 0.809 s  <<< ERROR!
org.jboss.weld.exceptions.DeploymentException: 
WELD-001408: Unsatisfied dependencies for type MultipleEntityRepo with qualifiers @Default
  at injection point [BackedAnnotatedField] @Inject ee.jakarta.tck.data.standalone.entity.EntityTests.shared
  at ee.jakarta.tck.data.standalone.entity.EntityTests.shared(EntityTests.java:0)

I don't see where a `shared` field is defined, and I also don't see a `MultipleEntityRepo` type anywhere.

--- a/runner/src/test/java/org/hibernate/ee/jakarta/data/tck/StandaloneEntityTests.java
+++ b/runner/src/test/java/org/hibernate/ee/jakarta/data/tck/StandaloneEntityTests.java
@@ -10,6 +10,7 @@ import ee.jakarta.tck.data.framework.read.only.NaturalNumbers_;
 import ee.jakarta.tck.data.framework.read.only.PositiveIntegers;
 import ee.jakarta.tck.data.framework.read.only.PositiveIntegers_;
 import ee.jakarta.tck.data.standalone.entity.Boxes_;
+import ee.jakarta.tck.data.standalone.entity.MultipleEntityRepo_;
 import ee.jakarta.tck.data.standalone.entity.EntityTests;
 import jakarta.data.repository.Repository;
 import jakarta.enterprise.context.RequestScoped;
@@ -50,6 +51,7 @@ public class StandaloneEntityTests extends EntityTests {
                     NaturalNumbers_.class,
                     PositiveIntegers_.class,
                     CustomRepository_.class,
+                    MultipleEntityRepo_.class,
                     DirectoryBuildCompatibleExtension.class)
             .activate(RequestScoped.class)
             .setPersistenceUnitFactory((Function) new EMFAction())

--- a/runner/src/test/resources/META-INF/persistence.xml
+++ b/runner/src/test/resources/META-INF/persistence.xml
@@ -8,14 +8,24 @@
         <description>Hibernate Entity Manager for Jakarta Data TCK</description>
         <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
 
+        <class>ee.jakarta.tck.data.framework.read.only.AsciiCharacter</class>
+        <class>ee.jakarta.tck.data.framework.read.only.NaturalNumber</class>
+        <class>ee.jakarta.tck.data.standalone.entity.Box</class>
+        <class>ee.jakarta.tck.data.standalone.entity.Coordinate</class>
+        <class>ee.jakarta.tck.data.web.validation.Rectangle</class>
+
         <properties>
-            <property name="jakarta.persistence.jdbc.driver" value="org.h2.Driver" />
-            <property name="jakarta.persistence.jdbc.url"    value="jdbc:h2:./target/h2temp;DB_CLOSE_DELAY=-1" />
-            <property name="jakarta.persistence.jdbc.user"   value="sa" />
+            <property name="jakarta.persistence.jdbc.url"
+                      value="jdbc:h2:./target/h2temp;DB_CLOSE_DELAY=-1" />
+            <property name="jakarta.persistence.jdbc.user" value="sa" />
             <property name="jakarta.persistence.jdbc.password" value="" />
-            <property name="hibernate.hbm2ddl.auto" value="create-drop"/>
+
+            <property name="jakarta.persistence.schema-generation.database.action"
+                      value="drop-and-create"/>
+
             <property name="hibernate.show_sql"   value="true" />
             <property name="hibernate.format_sql" value="true" />
+            <property name="hibernate.highlight_sql" value="true" />
         </properties>
     </persistence-unit>
 </persistence>

--- a/runner/src/test/resources/META-INF/persistence.xml
+++ b/runner/src/test/resources/META-INF/persistence.xml
@@ -23,6 +23,8 @@
             <property name="jakarta.persistence.schema-generation.database.action"
                       value="create"/>
 
+            <property name="hibernate.allow_update_outside_transaction" value="true"/>
+
             <property name="hibernate.show_sql"   value="true" />
             <property name="hibernate.format_sql" value="true" />
             <property name="hibernate.highlight_sql" value="true" />

--- a/runner/src/test/resources/META-INF/persistence.xml
+++ b/runner/src/test/resources/META-INF/persistence.xml
@@ -21,7 +21,7 @@
             <property name="jakarta.persistence.jdbc.password" value="" />
 
             <property name="jakarta.persistence.schema-generation.database.action"
-                      value="drop-and-create"/>
+                      value="create"/>
 
             <property name="hibernate.show_sql"   value="true" />
             <property name="hibernate.format_sql" value="true" />

